tools/depends/target/python3/darwin_embedded.patch

```diff
@@ -277,7 +278,7 @@
      if (pid == 0) {
          /* child: this clobbers and resets the import lock. */
          PyOS_AfterFork_Child();
-@@ -5447,7 +5473,7 @@
+@@ -5444,7 +5470,7 @@
```

Comment:
Is the current range information incorrect?

Reply:
Yes